lms/djangoapps/shoppingcart/models.py

```diff
@@ -1939,12 +1941,17 @@ def additional_instruction_text(self):
                 "If you haven't verified your identity yet, please start the verification process ({verification_url})."
             ).format(verification_url=verification_url)
 
+        if is_professional_mode_verified:
+            refund_reminder_msg = _("You have only until two days after the course starts to unenroll and receive a "
```

Comment:
You can unenroll in the course and receive a full refund for 2 days after the course start date.

(Since this timeline is so short, should we say "48 hours" instead?)